### PR TITLE
A few edits for the telemetry page

### PR DESCRIPTION
**Pull request by ganicke**

### CODE REVIEW

File: docs/telemetry.md [hunk 1]
```diff
@@ -57,10 +65,10 @@ Property | Description
 `user.storeRegion` |
 
 
-### User Session telemetry
+#### User Session telemetry
 
 A session represents a series of user actions. A session starts with a user action.
-It ends at the last user activity when there is no more activity for 30 minutes, or if it lasts longer than 30 minutes.
+It ends at the last user action when there's been no further activity for 30 minutes, or if it lasts longer than 30 minutes.
```
Comment (ganicke) [inline docs/telemetry.md]: I don't understand "...or if it lasts longer than 30 minutes.". Do you count just 30 minutes or 30 minutes plus the activity delta?

Comment (abchatra) [reply docs/telemetry.md]: Sessions expire on 30minutes of inactivity or after 24hours. The prior wordings seems just wrong.

File: docs/telemetry.md [hunk 2]
```diff
@@ -57,10 +65,12 @@ Property | Description
...
 
-### User Session telemetry
+#### User Session telemetry
 
-A session represents a series of user actions. A session starts with a user action.
-It ends at the last user activity when there is no more activity for 30 minutes, or if it lasts longer than 30 minutes.
+A session represents a series of user actions. A session starts with a user action. Sessions expire on either of these conditions:
+
+* 30 minutes of inactivity.
+* After 24 hours while any period of inactivity is less than 30 minutes.
```
Comment (abchatra) [inline docs/telemetry.md]: After 24 hours sessions expire, irrespective of activity or inactivity of less than 30 minutes

Comment (abchatra) [reply docs/telemetry.md]: In other words max duration of session is 24 hours.

**Pull request merged**